### PR TITLE
migrated issue templates to new github format

### PR DESCRIPTION
**Pull request by ghuntley**
## PR Type
What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

User has to manually customise a generic template

## What is the new behavior?

User is provided with options, depending on which option they choose they get a specialised template and GitHub will automatically add the appropriate labels to the issue.

new-issue-page-with-multiple-templates

See https://help.github.com/en/articles/about-issue-and-pull-request-templates

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+name: Bug Report
+about: Report a bug encountered while developing with Uno
+labels:
+- kind/bug
+- triage/untriaged
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://github.com/nventive/Uno/security/
```
Comment (davidjohnoliver) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Went to the link, looks like there's some setup required on our side

Comment (ghuntley) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Raised and tracking over at https://github.com/nventive/Uno/pull/976

**Pull request merged**